pocketimdb/movies/views.py

```diff
@@ -2,10 +2,12 @@
 from rest_framework.permissions import IsAuthenticated
 from .models import Movie
 from .serializers import MovieSerializer
+from .paginations import CustomPageNumberPagination
 
 class MovieViewSet(mixins.ListModelMixin,
                 viewsets.GenericViewSet):
 
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
-    permission_classes = [IsAuthenticated]
\ No newline at end of file
+    permission_classes = [IsAuthenticated]
+    pagination_class = CustomPageNumberPagination
```

Comment:
ovo takodje mozemo postaviti na nivou citavgo projekta, jer kada radis jednom paginaciju uglavnom ces na svim mjestima isti princip koristi, neces mjenjati nacin vracanja response, tako da mozes na nivou settings-a definsitati ovo. https://www.django-rest-framework.org/api-guide/pagination/#setup

Reply:
U redu, ubacicu onda u settings. Nekako mi je razmisljanje bilo da zasebno da movies samo definisem, mada svejedno nemam endpoint za listu usera tako da mi u ovom slucaju to ne menja mnogo, ali stavljam u settings onda svakako